src/components/Collaborative/Form/SourceCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { uniqueId } from "remirror";
+import { Col } from "antd";
+import CardStyle from "./CardStyle";
+import { useTranslation } from "next-i18next";
+
+export const SourceCard = ({ forwardRef }) => {
```

Comment:
question: can we create a more generic component for the Editor inputs? It seems that we have a very similar structure by now that could be converted to a Factory.